easybuild/easyconfigs/r/Rosetta/Rosetta-3.5-ictce-5.4.0.eb

```diff
@@ -0,0 +1,19 @@
+name = 'Rosetta'
+version = '3.5'
+
+homepage = 'https://www.rosettacommons.org'
+description = """Rosetta is the premier software suite for modeling macromolecular structures. As a flexible,
+multi-purpose application, it includes tools for structure prediction, design, and remodeling of proteins and
+nucleic acids."""
+toolchain = {'name': 'ictce', 'version': '5.4.0'} 
+toolchainopts = {'usempi': True}
+
+sources = ['%(namelower)s%(version)s_bundles.tgz']
+
+modextravars = {
+    'ROSETTA3_DB': '$root/rosetta_database'
```

Comment:
can you add this in the two existing easyconfigs for Rosetta too?

also, minor remark: add a `,` at the end of this line please :-)